app/controllers/users_controller.rb

```diff
@@ -65,6 +65,8 @@ def create
   def create_external
     @user = User.new(params[:user])
     @user.password = generate_new_password
+    @user.created_by = current_user.try(:id)
```

Comment:
I'm not a fan of our existing relationships where `created_by` is an ID, rather than a `belongs_to`. I would lean towards changing the column to `created_by_id` and adding a `belongs_to`.